### PR TITLE
feat: create form component create/edit

### PR DESCRIPTION
**Pull request by aisyahnnd**
https://www.loom.com/share/455bd686336c4878a28db874d01576b8

### CODE REVIEW

File: src/Parts/MyTrainingCard/SingleTrainingCard.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+/* eslint-disable react/prop-types */
```
Comment (argorfc) [inline src/Parts/MyTrainingCard/SingleTrainingCard.jsx]: Please remove the fixing comment and make sure use current linter

**Review by argorfc — CHANGES_REQUESTED** [review]
Read the comment.

File: src/Parts/MyTrainingCard/SingleTrainingCard.jsx (hunk 1 above)
Comment (aisyahnnd) [reply src/Parts/MyTrainingCard/SingleTrainingCard.jsx]: already fix eca2dd60b6bc983ef310fad67efbb388a63ba38f

**Review by argorfc — APPROVED** [review]
Approved.
Ready to merge.

**Pull request merged**